Solutions/GoogleCloudPlatformDNS/Hunting Queries/GCPDNSRareDomains.yaml

```diff
@@ -0,0 +1,28 @@
+id: 49d88918-03c8-4e22-9c8b-843e79cad6f5
+name: Google DNS - Rare domains
+description: |
+  'Query searches for requests rare domains.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: GCPDNSDataConnector
+    dataTypes:
+      - imDNSGCPCloudDNS
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1095
+query: |
+  imDNSGCPCloudDNS
+  | where TimeGenerated > ago(24h)
+  | summarize count() by Query
+  | order by count_ asc

```

Comment:
How are we determining which are rare. Wondering if we wanted to use
top 5 or top 10 here.

Reply:
fixed